core/lib/config/src/configs/eth_sender.rs

```diff
@@ -22,7 +22,7 @@ impl EthConfig {
     pub fn for_tests() -> Self {
         Self {
             sender: Some(SenderConfig {
-                aggregated_proof_sizes: vec![1, 4],
+                aggregated_proof_sizes: vec![1],
```

Comment:
It was because we only used 4 for dummy proofs on testnet. However, since solpp removal, the contracts have changed a bit on testnet, and we can just submit a single proof at a time. Note that this was always true on mainnet: 
https://github.com/matter-labs/era-contracts/blob/4552d5227d5a9663c2b4431830771b7ab734e4ad/l1-contracts/contracts/state-transition/chain-deps/facets/Executor.sol#L439